### PR TITLE
Add page intro responsive styles

### PR DESCRIPTION
**Pull request by Saionaro**

### CODE REVIEW

File: components/system.ts [hunk 1]
```diff
@@ -51,6 +51,7 @@ export interface StyledSystemProps
   textDecoration?: ResponsiveValue<Property.TextAlign, RequiredTheme>;
   textTransform?: ResponsiveValue<Property.TextTransform, RequiredTheme>;
   listStyle?: ResponsiveValue<Property.ListStyle, RequiredTheme>;
+  borderCollapse?: ResponsiveValue<Property.BorderCollapse, RequiredTheme>;
```
Comment (iAdramelk) [inline components/system.ts:54]: Did you miss also adding it to the `system` below or it works without it?

File: components/Layout/Centrator.tsx [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
 export function Centrator({ children, as = "div", ...props }: FlexProps) {
   return (
-    <Flex px="11" justifyContent="center" width="100%">
-      <Flex as={as} maxWidth={1240} {...props}>
+    <Flex
+      px="11"
+      justifyContent="center"
+      width="100%"
+      css={media("md", {
+        px: 3,
```
Comment (iAdramelk) [inline components/Layout/Centrator.tsx:11]: Maybe it's easier to use `[3,3,0]` here?

**Review by iAdramelk — COMMENTED** [review]
P. S. Can you add screenshots of the components to the PRs? Its easier to check them this way.

File: components/system.ts (hunk 1 above)
Comment (Saionaro) [reply components/system.ts:54]: Actually I forget about it! But it works even without adding :|

File: components/Layout/Centrator.tsx (hunk 2 above)
Comment (Saionaro) [reply components/Layout/Centrator.tsx:11]: Wow, it's great

**Pull request merged**